src/lib/types/model.ts

```diff
@@ -108,6 +108,10 @@ export interface ISortOrder {
     [index: string]: number;
 }
 
+export interface ICustomHeaders {
+    [name: string]: string;
+}
+
```

Comment:
Ok, I think you had the idea of doing something as I suggested, but I believe this is not used right now

Reply:
You're right. I forgot to remove this one.